runtime/doc/repeat.txt

```diff
@@ -788,6 +788,19 @@ DEFINING BREAKPOINTS
 <		Note that this only works for commands that are executed when
 		sourcing the file, not for a function defined in that file.
 
+:breaka[dd] expr {string}
+		Sets a breakpoint, that will break whenever the {string}
+		evaluates to true. Example: >
+			:breakadd expr g:lnum
+
+<		Will break, whenever the global variable lnum changes.
```

Comment:
According to the first sentence this should break each time `g:lnum` happens to be `1`, not when this changes. Something needs to be fixed.